bootstrap/012_task_log.sql

```diff
@@ -0,0 +1,119 @@
+
+create or replace function internal.root_task_id()
+returns text
+AS
+$$
+     SYSTEM$TASK_RUNTIME_INFO('CURRENT_ROOT_TASK_UUID')
+$$;
+
+create or replace function internal.task_run_id()
+returns text
+AS
+$$
+     SYSTEM$TASK_RUNTIME_INFO('CURRENT_TASK_GRAPH_RUN_GROUP_ID')
+$$;
+
+-- Inserts a row into TASK_LOG with the time the task started, the name of the object being materialized, the graph run ID and query_id for the task.
+create or replace procedure internal.start_task(task_name text, object_name text, start_time timestamp_ltz, task_run_id text, query_id text)
```

Comment:
just noticed that passing a `timestamp_ltz` through a procedure seems to is shifting by the account's default timezone. passing as text doesn't do this.

Reply:
let's use timestamp_ntz everywhere.

Reply:
ok